content/docs/concepts/traffic-management/routing/gateways/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Gateways
+description: Learn about using gateways to configure how the Envoy proxies load balance HTTP, TCP, or gRPC traffic.
+weight: 3
+keywords: [traffic-management, http, tcp, grpc, ingress, egress]
+---
+
+You use a [gateway](/docs/reference/config/networking/v1alpha3/gateway/)
+to manage inbound and outbound traffic for your mesh. You can manage the
+following types of traffic with a gateway: HTTP TCP gRPC
```

Comment:
Why are you listing these 3 protocols?
https://preliminary.istio.io/docs/reference/config/networking/v1alpha3/gateway/#Port

Reply:
Because I didn't remember we had all ports listed there. Thanks.